pages/auth/register/applicant/index.html

```diff
@@ -1,49 +1,91 @@
-            <div class="error"></div>
-          </div>
-          <div class="col-md-6">
-            <label class="mb-2" for="emailStudent">Email*</label>
-            <input type="email" class="form-control" aria-label="Email" name="email" id="emailStudent" pattern="^[\w\-.]+@(stud.)?noroff.no$" title="The email value must be a valid stud.noroff.no or noroff.no email address." required>
-            <div class="error"></div>
-          </div>
-          <div class="col-md-6">
-            <label class="mb-2" for="passwordStudent">Password*</label>
-            <input type="password" class="form-control" aria-label="Password" name="password" id="passwordStudent" minlength="8" title="The password value must be at least 8 characters." required>
-            <div class="error"></div>
-          </div>
-          <div class="col-md-6">
-            <label class="mb-2" for="repPasswordStud">Repeat password*</label>
-            <input type="password" class="form-control" aria-label="Repeat password" minlength="8" title="The password value must be at least 8 characters." id="repPasswordStud" required />
-            <div class="error"></div>
-          </div>
-          <div class="col-md-6">
-            <label class="mb-2" for="degrees">Degree(s)</label>
-            <select class="form-select" aria-label="Default select example">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!-- Description -->
+  <meta name="description"
+    content="This page lets users who are looking for work to register as Applicants on the website." />
+  <title>Noroff Jobs</title>
+  <!-- Links and scripts -->
+  <link href="/css/index.css" rel="stylesheet" />
+  <script src="/index.js" defer type="module"></script>
+</head>
+
+<body id="registerPageApplicant">
+  <header></header>
+  <main class="px-sm-3 py-5 bg-theme-dark">
+    <div class="container maxWidthContainer my-3 shadow bg-white p-3 rounded-1 w-50" style="min-width: 300px;">
+      <h1 class="fw-bold">Register For Applicant</h1>
+      <!-- Register form -->
+      <form class="row g-3" id="registerForm-applicant">
+        <div class="d-flex justify-content-center mb-3">
+          <div class="d-flex justify-content-center align-items-center rounded-circle bg-theme-new-grey registerImg">
+            <button type="button" class="btn text-decoration-underline" data-bs-toggle="modal"
+              data-bs-target="#exampleModal">Add image</button></div>
+        </div>
+        <div class="col-md-6">
+          <label class="mb-2" for="fullName">Full Name*</label>
+          <input type="text" class="form-control" aria-label="Full name" id="fullName" required
+            pattern="^[a-zA-ZæøåÆØÅ]+( [a-zA-ZæøåÆØÅ]+)+$"
```

Comment:
I suppose this is the only thing you changed right? I see a bunch of changes have been made, but looks to me like it's just formatting changes?

Reply:
yes, that was the only thing I changed, the other changes happened automatically, so I expect it to be formatting changes, I think prettier would have spaces between some elements, I'm not sure if it's something I have globally on the PC or on the project.